client/components/sub-masterbar-nav/style.scss

```diff
@@ -3,7 +3,7 @@
 	display: flex;
 	flex-direction: column;
 	padding: 56px 20px 28px;
-	background-color: $masterbar-color;
+	background-color: $masterbar-bg-color;
```

Comment:
Good point! While I used `bg` here in order to stay consistent with the already existing `$sidebar-bg-color`, I do agree that in both cases we probably want to use the full words.